### PR TITLE
Feature/common_widgets/Bank_card_view

### PR DESCRIPTION
**Pull request by MilenniyamAlbiorix**

### CODE REVIEW

File: lib/src/presentation/core/utils/widgets/custom_bank_card_view.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    required this.onSelected,
+    this.imageWidth,
+    this.imageHeight,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: onSelected,
+      child: AnimatedContainer(
```
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/widgets/custom_bank_card_view.dart:26]: You haven't used any size changes. could you please explain the purpose of the animated container?

**Review by SohanAlbiorix — COMMENTED** [review]
Also, this is the stateless widget and if it is used in BLoC, does it rebuild when any state is updated or only the backgrounds and images are changed?

File: lib/src/presentation/core/utils/widgets/custom_bank_card_view.dart (hunk 1 above)
Comment (MilenniyamAlbiorix) [reply lib/src/presentation/core/utils/widgets/custom_bank_card_view.dart:26]: When I select another card I get the card blinking so I use the animated container

**Comment by MilenniyamAlbiorix** [comment]
> Also, this is the stateless widget and if it is used in BLoC, does it rebuild when any state is updated or only the backgrounds and images are changed?

**Comment by MilenniyamAlbiorix** [comment]
> > Also, this is the stateless widget and if it is used in BLoC, does it rebuild when any state is updated or only the backgrounds and images are changed?

**Pull request merged**